linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_clients.js

```diff
@@ -52,15 +65,15 @@ var RouterClient = (function() {
     return summary;
   }
 
-  function timeseriesParams() {
+  function timeseriesParams(name) {
     return {
-      strokeStyle: colorLookup[colorIdx++ % colorLookup.length], //TODO: #242 per client color scheme
+      strokeStyle: chartLegend[name.replace("request_latency_ms.", "")],
```

Comment:
I'm getting the metrics here from client.metrics (from routers.js), which just returns "request_latency_ms.max: 0" etc...  So the client name isn't in name in this case.  

I'm not actually sure whether I should be using client.metrics from routers or if I should be getting latency from metrics.json.